python/pyspark/sql/types.py

```diff
@@ -580,7 +581,7 @@ def scalaUDT(cls):
         """
         The class name of the paired Scala UDT.
```

Comment:
Should we update the comment to clarify that we return an empty string if there is no corresponding Scala UDT?